src/pep8_naming/checks.rs

```diff
@@ -224,7 +224,7 @@ pub fn error_suffix_on_exception_name(
 ) -> Option<Check> {
     if bases.iter().any(|base| {
         if let ExprKind::Name { id, .. } = &base.node {
-            id == "Exception"
+            id.ends_with("Exception") || id.ends_with("Error")
```

Comment:
Do we want to exempt `BaseException`, maybe with `id == "Exception" || id.ends_with("Error")`?

Reply:
https://github.com/charliermarsh/ruff/pull/798